controller/volume_controller.go

```diff
@@ -1405,6 +1410,19 @@ func (c *VolumeController) ReconcileVolumeState(v *longhorn.Volume, es map[strin
 	return c.checkAndFinishVolumeRestore(v, e, rs)
 }
 
+func (c *VolumeController) checkVolumeMountFileSystemStatus(v *longhorn.Volume, e *longhorn.Engine) (err error) {
+	log := getLoggerForVolume(c.logger, v)
+	if v.Status.State == longhorn.VolumeStateAttached && e.Status.CurrentState == longhorn.InstanceStateRunning {
+		if e.Status.FilesystemReadOnly {
+			v.Status.RemountRequestedAt = c.nowHandler()
+			log.Infof("Volume request remount at %v", v.Status.RemountRequestedAt)
+			msg := fmt.Sprintf("Volume %v requested remount at %v", v.Name, v.Status.RemountRequestedAt)
+			c.eventRecorder.Eventf(v, corev1.EventTypeNormal, constant.EventReasonRemount, msg)
+		}
+	}
+	return nil
+}
```

Comment:
Consider removing the error return from this method since it will always return nil.

Reply:
fixed, thanks!